setup.sh

```diff
@@ -1,18 +1,24 @@
 #!/bin/sh
 
 uninstall_plaster() {
+    bin_root="$1"
+    share_root="$2"
+
     echo "Uninstalling current installation of plaster"
-    sudo rm /usr/bin/plaster;
-    sudo rm -rf /usr/share/plaster;
+    sudo rm ${bin_root}/plaster;
+    sudo rm -rf ${share_root}/plaster;
 }
 
 install_plaster() {
+    bin_root="$1"
+    share_root="$2"
+
     echo "Building project"
     mvn clean install -DskipTests=true -q;
 
     echo "Installing plaster"
-    sudo mkdir /usr/share/plaster;
-    sudo mkdir /usr/share/plaster/lib;
+    sudo mkdir ${share_root}/plaster;
```

Comment:
Just add the plaster to the `share_root` and make it `share_location`. Do the same with `bin_root`. That way we know we are using the same location in `uninstall_plaster` as well